archive/h/haskell/file-io.hs

```diff
@@ -0,0 +1,17 @@
+module Main where
+
+import System.IO
+
+main :: IO ()
+main = do
+  -- write
+  handle <- openFile "file.txt" ReadWriteMode
```

Comment:
Looks good! For consistency, can you make the file `output.txt`. I believe that's what's specified in the documentation.

Reply:
@jrg94 I made that change. Sorry I missed that in the docs.

Reply:
No worries! This is why we have code reviews. :)